src/index.html

```diff
@@ -22,6 +22,468 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="index.html"
+        class="logo-link"
+      >
+        <img
+          src="./images/logo.png"
+          alt="Moyo logo"
+        />
+      </a>
+      <nav class="nav">
+        <ul class="nav__links">
+          <li class="nav__link-item">
+            <a
+              class="nav__link is-active"
+              href="/apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              data-qa="nav-hover"
+              class="nav__link"
+              href="/laptops"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__link-item">
+            <a
+              class="nav__link"
+              href="/video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="products">
+      <div
+        class="card"
+        data-qa="card"
```

Comment:
Each card should have a unique `data-qa` attribute value if they are meant to be identified separately in tests.